src/main/java/lesson01/loopspracticing/BubbleSort.java

```diff
@@ -11,14 +11,13 @@
     public static void main (String[] args) {
 
         System.out.println("--------1.2 - Bubble sorting algorithm--------");
-        List<Integer> createdArrayForSorting =  ArrayGenerating.generateRandomArrayList(20);
+        List<Integer> createdArrayForSorting =  lesson01.arraysmanipulation.ArrayGenerator.generateRandomArrayList(20);
```

Comment:
I'd recommend adding a link to class like import lesson01.arraysmanipulation.ArrayGenerator;